pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -215,6 +215,16 @@ spec:
               protocol: TCP
             - containerPort: 3232
               protocol: TCP
+            livenessProbe:
+              exec:
+                command:
+                - /bin/sh
+                - -c
+                - zfs set io.openebs:livenesstimestap='$(date)' cstor-$OPENEBS_IO_CSTOR_ID
+              failureThreshold: 3
+              initialDelaySeconds: 300
+              periodSeconds: 10
+              timeoutSeconds: 30
```

Comment:
Is there a possibility of a clash between periodSeconds and timeoutSeconds? For instance, the current probe is not yet timed-out and the next one has started.

Reply:
No. Until the probe finishes either with a success or failure case next probe should not start.

Reply:
as per the doc, it starts another probe @avishnu as you expected. After `timeoutSeconds`, if no response comes, another probe gets started after 10 seconds (i.e., 40 seconds after first probe). This happens for 3 times, and, if the 3rd one also fails, pod gets restarted. So, you can assume - (10+30)*3 seconds = 120 seconds to identify for pod to be marked as bad in case of any IO hang.

Reply:
Ok